lua/entities/gmod_wire_explosive.lua

```diff
@@ -5,6 +5,8 @@ ENT.WireDebugName 	= "Explosive"
 
 if CLIENT then return end -- No more client
 
+local wire_explosive_delay = CreateConVar( "wire_explosive_delay", game.SinglePlayer() and 0 or 0.2 )
```

Comment:
What if you make an SP server and then create a listen server?
Better make that singe player check on spawn